index.php

```diff
@@ -1,9 +1,22 @@
 <!DOCTYPE html>
-<html>
+<!--[if lt IE 7]>
+<html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>
+<html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>
+<html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!-->
+<html class="no-js"> <!--<![endif]-->
 <head>
-    
+    <base href="../">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <title>inrupt Atomic Design System</title>
```

Comment:
Seems strange to have two different bit of language for the titles here - "inrupt Atomic Design System vs. Solid Style Guide". I think we should be consistent.

Also inrupt is not capitalized unless at the beginning of a sentence or statement.

Reply:
@justinwb That's the old title that I removed, I had replaced it with:

<title>Solid Style Guide - by inrupt</title>

Reply:
lets remove the hyphen - because when it is referenced in other places we wouldn't want them to use that. So just "Solid Style Guide by inrupt"

Reply:
should be good to go!